lib/cli/parse.rs

```diff
@@ -616,53 +597,113 @@ pub(super) fn parse_payment_info(params: PaymentStrParams) -> Result<ExecutableD
     })
 }
 
-fn parse_contract_hash(value: &str) -> Result<Option<HashAddr>> {
+fn contract_hash(value: &str) -> Result<Option<HashAddr>, CliError> {
```

Comment:
Nit: I prefered the old name of `parse_contract_hash` is there a reason we can't go back to that name?

Reply:
It's in line with all the other methods in this module.  The module is named `parse` so calling it looks like
```rust
parse::contract_hash
```
which avoids the stutter if we were to keep the old name.